pkg/acsengine/const.go

```diff
@@ -102,29 +103,7 @@ var KubeImages = map[string]map[string]string{
 		"ratelimitqps":    strconv.FormatFloat(DefaultKubernetesCloudProviderRateLimitQPS, 'f', -1, 64),
 		"ratelimitbucket": strconv.Itoa(DefaultKubernetesCloudProviderRateLimitBucket),
 	},
-	api.Kubernetes170: {
-		"hyperkube":       "hyperkube-amd64:v1.7.0",
-		"dashboard":       "kubernetes-dashboard-amd64:v1.6.1",
-		"exechealthz":     "exechealthz-amd64:1.2",
-		"addonresizer":    "addon-resizer:2.0",
-		"heapster":        "heapster:v1.4.0",
-		"dns":             "k8s-dns-kube-dns-amd64:1.14.4",
-		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
-		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.14.4",
-		"pause":           "pause-amd64:3.0",
-		"windowszip":      "v1.7.0intwinnat.zip",
```

Comment:
When we have v1.7.1, we don't need to have v1.7.0 anymore. This is the whole point of only maintaining one patched version of a minor. 

Yes, you would expect to see i removed quite a few code without adding back.